recipes/daw_json_link/all/conanfile.py

```diff
@@ -42,9 +42,12 @@ def requirements(self):
         if Version(self.version) < "3.23.2":
             self.requires("daw_header_libraries/2.97.0")
             self.requires("daw_utf_range/2.2.3")
-        else:
+        elif Version(self.version) < "3.24.1":
             self.requires("daw_header_libraries/2.101.0")
             self.requires("daw_utf_range/2.2.4")
+        else:
```

Comment:
Maybe we can do here as I exemplified [here](https://github.com/conan-io/conan-center-index/pull/24485#discussion_r1661902521)? wdyt @toge?